src/In.ProjectEKA.HipService/Discovery/Patient/Ranker/Meta.cs

```diff
@@ -1,8 +1,8 @@
-using System;
-
-namespace hip_service.Discovery.Patient.Ranker
+namespace In.ProjectEKA.HipService.Discovery.Patient.Ranker
 {
-    public struct Meta: IEquatable<Meta>
+    using System;
+
+    public struct Meta : IEquatable<Meta>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This struct overrides 'GetHashCode' and should therefore also override 'Equals'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4839081)

Reply:
https://github.com/ProjectEKA/hip-service/issues/51